pyuvdata/tests/test_mwa_corr_fits.py

```diff
@@ -0,0 +1,192 @@
+# -*- mode: python; coding: utf-8 -*
+# Copyright (c) 2019 Radio Astronomy Software Group
+# Licensed under the 2-clause BSD License
+
+"""Tests for MWACorrFITS object.
+
+"""
+
+import pytest
+import os
+import numpy as np
+
+from pyuvdata import UVData
+from pyuvdata.data import DATA_PATH
+import pyuvdata.tests as uvtest
+from astropy.io import fits
+
+# set up MWA correlator file list
+testdir = os.path.join(DATA_PATH, 'mwa_corr_fits_testfiles/')
+
+testfiles = ['1131733552.metafits', '1131733552_20151116182537_mini_gpubox01_00.fits',
+             '1131733552_20151116182637_mini_gpubox06_01.fits', '1131733552_mini_01.mwaf',
+             '1131733552_mini_06.mwaf', '1131733552_mod.metafits']
+filelist = [testdir + i for i in testfiles]
+
```

Comment:
I think it'd be nice to have the cotter comparison in a unit test if possible. Could we get a Cotter file for this same data and make a unit test that tests that objects constructed from them are the same as those constructed for this data?